config/rabbitmq.conf

```diff
@@ -44,3 +44,7 @@ log.console.level = info
 ## Log level to use when logging to the amq.rabbitmq.log exchange
 ##
 # log.exchange.level = info
+
+## Set consumer timeout high, since some tasks can be very slow.
+## This is 10 hours in milliseconds
+consumer_timeout = 36000000
```

Comment:
Not sure if the newline at EOF is enforced by any style guide or whatever, but I figured I'd mention it.